docs/core/install/includes/linux-install-70-dnf.md

```diff
@@ -0,0 +1,28 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 11/15/2022
+ms.topic: include
+---
+
+### Install the SDK
```

Comment:
I might be reading this wrong, but we only refer to this doc (instead of 6.0) when the OS might have multiple .NET versions. Should we tell users how to install .NET 6 here too?

Reply:
This include file has generic steps for installing 7 via dnf. This file is referenced in the linux-rhel article. To keep the articles short and sweet, we only include the latest release's instructions.